app/src/main/res/xml/single_blind_preferences.xml

```diff
@@ -33,4 +33,16 @@
         andriod:key="calibration"
         andriod:title="@string/settingsCalibration" />
 
+    <Preference
+        andriod:fragment="com.helio.app.ui.blind.SensorSettingsFragment"
+        andriod:key="sensor1"
+        andriod:title="sensor1"/>
+
+    <Preference
+        andriod:fragment="com.helio.app.ui.blind.SensorSettingsFragment"
+        andriod:key="sensor2"
+        andriod:title="sensor2"/>
+
+
+
```

Comment:
I guess these are for testing, but you should remove them. They also crash the app when you click on them for some reason.